### PR TITLE
fix(prejoin): Fix join without audio

### PR DESCRIPTION
**Pull request by vp8x8**

### CODE REVIEW

File: react/features/base/config/actions.js [hunk 1]
```diff
@@ -10,6 +10,24 @@ import { CONFIG_WILL_LOAD, LOAD_CONFIG_ERROR, SET_CONFIG } from './actionTypes';
...
 import { setConfigFromURLParams } from './functions';
 
+
+/**
+ * Updates the config with new options.
+ *
+ * @param {Object} options - The new options (to add).
+ * @returns {Function}
+ */
+export function amendConfig(options: Object) {
```
Comment (saghul) [inline react/features/base/config/actions.js:20]: Search for `_UPDATE_CONFIG`. We have this machinery already, albeit in a bit of a twisted way. If we want to make this a valid internal API we should consolidate it.

Comment (vp8x8) [reply react/features/base/config/actions.js]: Sorry, missed that.

I have made public the `updateConfig` action and used it.

Comment (saghul) [reply react/features/base/config/actions.js]: Don't be sorry, you couldn't have known!

File: react/features/prejoin/middleware.js [hunk 2]
```diff
@@ -24,6 +25,10 @@ MiddlewareRegistry.register(store => next => async action => {
         const state = getState();
         const { userSelectedSkipPrejoin } = state['features/prejoin'];
         const localVideoTrack = getLocalVideoTrack(state['features/base/tracks']);
+        const { options } = action;
+
+
```
Comment (saghul) [inline react/features/prejoin/middleware.js:30]: one too many new lines

Comment (vp8x8) [reply react/features/prejoin/middleware.js]: Fixed.

**Comment by vp8x8** [comment]
Jenkins please test this please.

**Comment by vp8x8** [comment]
@saghul Is there any way to re-trigger the tests?

**Comment by damencho** [comment]
This would be nice to have tests. I should have think of that earlier ... can you try adding tests for that.
https://github.com/jitsi/jitsi-meet-torture/blob/master/src/test/java/org/jitsi/meet/test/PreJoinTest.java
You can click on the button joining a conference, check that the notification is shown and that mic is muted and cannot be unmuted... ping me if you need help.

**Comment by damencho** [comment]
@vp8x8 ^

**Comment by vp8x8** [comment]
@damencho Sure. I'll write a test for it.

**Pull request merged**